data/human/human missions.txt

```diff
@@ -5894,7 +5894,7 @@ mission "Quicksilver Mixup 2"
 	on visit
 		dialog `You have reached <planet>, but your escort, loaded with the shipment, has not arrived. Better depart and wait for your escorts to arrive in this star system.`
 	on complete
-		dialog `You contact the spaceport authorities and they direct you to the respective landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to <origin>. Again, you have 10 days to make the delivery.`
+		dialog `You contact the spaceport authorities and they direct you to the right landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to <origin>. Again, you have 10 days to make the delivery.`
```

Comment:
```suggestion
		dialog `You contact the spaceport authorities and they direct you to the landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to <origin>. Again, you have 10 days to make the delivery.`
```

Reply:
I'm not sure why Tibe's suggestion here has been marked resolved; instead, the word "respective" has been changed to "right". I will say that "respective" reads odd as nobody is comparing landing pads, but just cutting the world altogether reads better, as changing it to "right" has the same issue where it suggests a comparison that doesn't exist.

Reply:
I see. It does get a little confusing when typos are done in separate PRs, but that is partly on me for being confused.